src/services/item/plugins/tag/errors.ts

```diff
@@ -0,0 +1,24 @@
+import { StatusCodes } from 'http-status-codes';
+
+import { ErrorFactory } from '@graasp/sdk';
+
+const PLUGIN_NAME = 'graasp-plugin-tags';
+
+/**
+ * Errors thrown by the chat tasks
+ */
+
+export const GraaspTagsError = ErrorFactory(PLUGIN_NAME);
+
+export class ItemTagAlreadyExists extends GraaspTagsError {
+  constructor(data?: unknown) {
```

Comment:
Instead of unknown, can we choose a shape for the data that will be given to the error ? This could be a first step to ensure we do not leak information without noticing.